datafusion-examples/examples/custom_datasource.rs

```diff
@@ -243,8 +243,8 @@ impl ExecutionPlan for CustomExec {
             db.data.values().cloned().collect()
         };
 
-        let mut id_array = UInt8Builder::new(users.len());
-        let mut account_array = UInt64Builder::new(users.len());
+        let mut id_array = UInt8Builder::new();
```

Comment:
`with_capacity` instead?

Reply:
I'm happy to put these all back to `with_capacity()`, but now that they take two parameters, I wasn't sure what to set the `data_size` to. (`item_count` is straight-forward).

Reply:
UInt8Builder is for primitive type UInt8, and its `with_capacity` takes only one parameter?
```rs
/// Creates a new primitive array builder with capacity no of items
    pub fn with_capacity(capacity: usize) -> Self {
```

Reply:
You're right - I just got sloppy after fixing a few dozen of these. This is resolved now.